### PR TITLE
Chapter 3 review and adjustments

### PR DESCRIPTION
**Pull request by ingydotnet**
By submitting this pull request, I confirm that you can use, modify, copy, and
redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by eemeli — APPROVED** [review]

File: spec/1.2.2/spec.md [hunk 1]
```diff
@@ -676,6 +676,7 @@ structure] in this format.
 Therefore, this specification defines two concepts: a class of data objects
 called YAML [representations] and a syntax for [presenting] YAML
 [representations] as a series of characters, called a YAML [stream].
+
```
Comment (Thom1729) [inline spec/1.2.2/spec.md:679]: Was this deliberate?

File: spec/1.2.2/spec.md [hunk 2]
```diff
@@ -4407,8 +4407,8 @@ Mapping entries are separated by a "`,`" character.
 * [ns-flow-map-entry(n,c)] <!-- one_:_two three:_four five:_six seven_:_eight -->
 
 
-If the optional _"`?`" mapping key indicator_ is specified, the rest of the
-entry may be [completely empty].
+If the optional "`?`" mapping key indicator is specified, the rest of the entry
```
Comment (Thom1729) [inline spec/1.2.2/spec.md:4410]:
```suggestion
If the optional "`?`" _mapping key indicator_ is specified, the rest of the entry
```

File: spec/1.2.2/spec.md (hunk 1 above)
Comment (ingydotnet) [reply spec/1.2.2/spec.md:679]: Yes. This is a paragraph split.

File: spec/1.2.2/spec.md (hunk 2 above)
Comment (Thom1729) [reply spec/1.2.2/spec.md:4410]: In general, keep the italics on the term, just remove it from the characters. This applies to the next few as well.

Comment (Thom1729) [reply spec/1.2.2/spec.md:4410]: Eh, never mind, it's fine without them.

**Review by Thom1729 — APPROVED** [review]

File: spec/1.2.2/spec.md (hunk 2 above)
Comment (ingydotnet) [reply spec/1.2.2/spec.md:4410]: ok

**Pull request merged**